package/gluon-mesh-vpn-wireguard/check_site.conf

```diff
@@ -0,0 +1,19 @@
+need_boolean(in_site({'mesh_vpn', 'wireguard', 'configurable'}), false)
```

Comment:
What does configurable represent?

Reply:
a leftover from the gluon-mesh-vpn-fastd package. removing.